src/app/RecordingMetadata/BulkEditLabels.tsx

```diff
@@ -190,9 +190,9 @@ export const BulkEditLabels: React.FunctionComponent<BulkEditLabelsProps> = (pro
         : context.target.target().pipe(
             filter((target) => target !== NO_TARGET),
             concatMap((target) =>
-              context.api.graphql<any>(`
-              query {
-                targetNodes(filter: { name: "${target.connectUrl}" }) {
+              context.api.graphql<any>(
+                `query GetActiveRecordingsForTarget($connectUrl: String) {
```

Comment:
I think some operation names do not have `Get` prefix. Maybe we should either remove it or add for others?

Reply:
Like `query AllTargetsArchives` does not have `Get`.